source/access/managing_disk_usage.rst

```diff
@@ -6,6 +6,22 @@ How much disk space am I using?
 
 .. _quota:
 
+Disk quota
+==========
+
+VSC file systems can have 2 types of quota set on them:
+
+* quota on the *disk space*
+* quota on the *number of inodes*
+
+The number of inodes determines the maximum number of files that can be present
+in the file system. The purpose of the inode quota is to maintain good I/O
```

Comment:
in the ugent tier2 setup and also on tier1 the total amount of inodes is finite like the regular storage. and this is the main reason why we have quota on them. it has nothing to do with performance.

Reply:
i'm not a file system expert so correct me if i'm wrong, but i don't think this is correct.

if you wanted could increase the number of inodes by decreasing the bytes per inode?
so why don't you do that? because of performance? because random file access is expensive?

Reply:
in our gpfs and lustre config, the inodes are hosted on ssd or nvme, not on the spinning disks. so they are finite, and complicated to extend only this storage. (well, as complicated as the non-inode storage; so we have the same reasons for inode quota as for regular quota).

changing the inode size is not trivial, there is only small (512byte) and large (4kB); and the large inodes actually hold very small files; eg in gpfs on tier2 files with size under 4k-few 100 bytes of inode are actually fully stored in the inode. but that is not really the point

Reply:
thank you for that info.

even so, you could have chosen a larger inode storage and small sizes from the start, so it's a design choice, and the reason for that choice is performance?

Reply:
performance has nothing to do with it. it's the same reason the regular quota exists: the resource is finite. ofcourse  it also helps to keep people from doing stupid things that might impact the whole system. 
but if people need insane amounts of small files, they can get them. but not by default, they will need to motivate why this is needed.

Reply:
it's not just helping. the quota are an important safeguard to limit the amount of random IO, which indeed impacts the whole system. 
of course this is not enough, users can still do stupid things, but that's exactly why i wrote this, to make users aware of this limitation.

anyway, i give up. can you please provide an alternative text so i can get this merged?